packages/fast-form-generator-react/src/form/form-section.style.ts

```diff
@@ -78,13 +78,18 @@ const styles: ComponentStyles<IFormSectionClassNameContract, {}> = {
         },
         "&[aria-pressed='true']": {
```

Comment:
not in your changeset, but this is going to throw that link error - can you change to `&[aria-pressed=\"true\"]` while you're in here?

Reply:
whoops yep